### PR TITLE
add deploy, rollback, and invoke --alias support. Closes #7

### LINKED ISSUE #7: Aliases
Provide commands for manually working with aliases. We don't recommend using aliases for environments, however they may still be useful in some cases, such as per-person aliases in a staging environment.

- [x] deploys
- [x] rollback
- [x] invoke
- [x] test
- [x] update tests so they pass
- [x] add tests
- [ ] maybe default flags to CurrentAlias so people can see that

### PR DESCRIPTION
**Pull request by tj**
Review please!

Adds `--alias name` to rollback, deploy, and invoke so that you can launch canary functions etc.

I'll squash that other lame commit after.

### CODE REVIEW

**Comment by mthenw** [comment]
What about `delete`? I would assume that `apex delete -a sth` would delete that alias.

File: cmd/apex/deploy/deploy.go [hunk 1]
```diff
@@ -39,12 +45,15 @@ func init() {
 
 	f := Command.Flags()
 	f.StringSliceVarP(&env, "env", "e", nil, "Environment variable")
+	f.StringVarP(&alias, "alias", "a", "", "Function alias")
```
Comment (mthenw) [inline cmd/apex/deploy/deploy.go]: I think it's a good idea to pass "current" as a default here.

**Comment by tj** [comment]
Hmm yea regarding delete I'm not sure if I would expect that to remove the alias or to remove the version specified by the alias. I think since multiple aliases could be pointing at the same version it might be best to leave that out for now

**Comment by tj** [comment]
squashed and added the default!

**Pull request merged**